server.js

```diff
@@ -61,9 +61,6 @@ function twoStatic(url, file) {
 }
 
 twoBrowserified("/katex.js", "katex", "katex");
-twoUse("/test/jasmine", express.static(path.dirname(
-    require.resolve("jasmine-core/lib/jasmine-core/jasmine.js"))));
-twoBrowserified("/test/katex-spec.js", "test/*[Ss]pec.js");
```

Comment:
I had a closer look and `jest-html-reporter` doesn't actually run the tests in the browser.  It only produces a HTML report that can be read afterwards.